packages/diagnostic/test/diagnose.test.ts

```diff
@@ -7,10 +7,10 @@ import { resolve } from 'path';
 import diagnose from '../src/diagnose';
 
 const basePath = resolve(__dirname, 'fixtures', 'diagnose');
-const filesNames = ['first.ts', 'react.tsx', 'second.ts'];
+const fileNames = ['first.ts', 'react.tsx', 'second.ts'];
 
 // Append basePath to file names
-const files = filesNames.map((file) => resolve(basePath, file));
```

Comment:
i dont see the diff here? maybe reset this file?

Reply:
filesNames => fileNames